tests/static/loadUtils.test.js

```diff
@@ -0,0 +1,198 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const intlTelInput = require("intl-tel-input");
+const { initPlugin, resetPackageAfterEach } = require("../helpers/helpers");
+require("../helpers/matchers");
+
+describe("loadUtils", function() {
```

Comment:
I added a little at the end of the test, but mostly this is just uncommenting and tweaking the syntax of the existing, commented-out tests.